README.md

```diff
@@ -79,7 +79,7 @@
 | 76 | Adithya Dutt Kambhampati (RA2011003011072) | [Link]() | 2 | |
 | 77 |	Arushi Roy (RA2011003011093) | [Link]() | 2 | |
 | 78 |	Mote Sarvesh Eknath (RA2011003011094) | [Link]() | 2 | |
-| 79 |	NIVEDITA ANAND (RA2011003011096) | [Link]() | 2 | |
+| 79 |	NIVEDITA ANAND (RA2011003011096) | [Link](https://github.com/niv-a/18CSE301J_RA2011003011096) | 2 |:white_check_mark: |
```

Comment:
@niv-a please provide the deployed GitHub pages link not the GitHub repository link 
```suggestion
| 79 |	NIVEDITA ANAND (RA2011003011096) | [Link](GitHub deployed link) | 2 |:white_check_mark: |
```